flytekit/core/base_task.py

```diff
@@ -462,6 +463,7 @@ def __init__(
         environment: Optional[Dict[str, str]] = None,
         disable_deck: Optional[bool] = None,
         enable_deck: Optional[bool] = None,
+        deck_fields: Optional[Tuple[DeckField, ...]] = (DeckField.SOURCE_CODE, DeckField.DEPENDENCIES),
```

Comment:
After speaking with @pingsutw and @EngHabu, we still need to keep the current default decks to keep backward compatibility. Can you update this to include all of them? (source code + dependencies + timeline + inputs + outputs).